core/src/main/scala/org/apache/spark/h2o/backends/SharedBackendConf.scala

```diff
@@ -164,15 +169,29 @@ trait SharedBackendConf {
 
   def setNodeBasePort(port: Int) = set(PROP_NODE_PORT_BASE._1, port.toString)
 
+  def setHadoopConf(path: String) = set(PROP_HADOOP_CONF._1, path)
+
+  def setHadoopConf(conf: Configuration): H2OConf = {
+    val hadoopConfigTempFile: File = File.createTempFile("hadoop_config_all", ".xml")
+    val fileWriter: FileWriter = new FileWriter(hadoopConfigTempFile)
+    try {
```

Comment:
nit: you can use `withResource` statement after you import `ai.h2o.sparkling.utils.ScalaUtils._`

Reply:
Done ! Great ! Thanks !